fili-core/src/main/java/com/yahoo/bard/webservice/data/PartialDataHandler.java

```diff
@@ -82,21 +40,21 @@ public SimplifiedIntervalList findMissingRequestTimeGrainIntervals(
      * present for a given combination of request metrics and dimensions (pulled from the API request and generated
      * druid query) at the specified granularity.
      *
-     * @param columnNames  all the column names the request depends on
+     * @param constraints  Constraint containing all the column names the request depends on
      * @param physicalTables  the tables whose column availabilities are checked
      * @param requestedIntervals  The intervals that may not be fully satisfied
      * @param granularity  The granularity at which to find missing intervals
      *
      * @return subintervals of the requested intervals with incomplete data
      */
     public SimplifiedIntervalList findMissingTimeGrainIntervals(
```

Comment:
We should leave this method as-is for now I think. We can come back to it later if needed.